docs/canonicalk8s/snap/reference/disa-stig-audit.md

```diff
@@ -60,13 +67,12 @@ accounts for each work payload
 >
 
 
-### Remediation
+Remediation
```

Comment:
I understand the reasoning for removing the header here, it does make for a nicer right navigation. However, I feel it does not look right in terms of the rest of the section. 

<img width="1076" height="743" alt="Image" src="https://github.com/user-attachments/assets/a352ce11-6589-48db-8844-a24b82936920" /> Could we bold the remediation and auditing as root to give them the same importance?